topics/microbiome/tutorials/dada-16S/tutorial.md

```diff
@@ -0,0 +1,1057 @@
+>
+> 2. {% tool [Phyloseq](interactive_tool_phyloseq) %} with the following parameters:
+>    - {% icon param-file %} *"Phyloseq R object"*: output of **Create phyloseq object from dada2**
+>
+>    {% snippet faqs/galaxy/interactive_tools_open.md %}
+>
+{: .hands_on}
+
+A new page will be launched with Shiny-phyloseq ready to explore the phyloseq object.
+
+## Visualize alpha-diversity
+
+A **diversity index** is a quantitative measure that is used to assess the level of diversity or variety within a particular system, such as a biological community, a population, or a workplace. It provides a way to capture and quantify the distribution of different types or categories within a system.
+
+Related to ecology, the term **diversity** describes the number of different species present in one particular area and their relative abundance. More specifically, several different metrics of diversity can be calculated. The most common ones are α, β and γ diversity.
+
+The **α diversity** describes the diversity within a community. It considers the number of different species in an environment (also referred to as species **richness**). Additionally, it can take the abundance of each species into account to measure how evenly individuals are distributed across the sample (also referred to as species **evenness**).
+
+There are several different indexes used to calculate α diversity because different indexes capture different aspects of diversity and have varying sensitivities to different factors. These indexes have been developed to address specific research questions, account for different ecological or population characteristics, or highlight certain aspects of diversity. 
+
+![Explanation of the α diversity](./images/alphadiversity_metrics.png "<a href="https://medium.com/pjtorres-high-gut-alpha-diversity-and-health/high-alpha-diversity-and-health-65e5eca7fa36">Source</a>")
+
+Here we will plot the **α diversity** for the samples given the Days using 2 measures:
+
+- The **Shannons** index, which calculates the uncertainty in predicting the species identity of an individual that is selected from a community ({% cite Shannon.1948 %}).
+
+- The **Simpsons** index, which calculates the probability that two individuals selected from a community will be of the same species. Obtains small values in datasets of high diversity and large values in datasets of low diversity ({% cite SIMPSON.1949 %}).
+
+> <hands-on-title> Plot α diversity </hands-on-title>
+>
+> 1. Click on **Alpha Diversity** tab
+> 2. Select the following parameters:
+>    - *"X"*: `Day`
+>    - *"Color"*: `When`
+>    - *"alpha Measures"*: `Shannon`, `Simpson`
+{: .hands_on}
+
+![2 panels for 2 α diversity metrics (Shannon, Simpson) with α diversity value on Y axis and days on X axis. The points are colored given When](./images/alpha_diversity.png)
+
+> <question-title></question-title>
+>
+> Is there any obvious systematic difference in alpha-diversity between early and late samples?
```

Comment:
Is there somewhere an example how an "obvious" difference would look like. Should we be clearer what we mean with difference? Smaller / larger?

Reply:
It would be nice to be clearer. Any suggestion there to rephrase that? I do not have inspiration tbh 😅

Reply:
Maybe just: 

```suggestion
> Is there any obvious systematic difference in alpha-diversity between early and late samples? That is is it obviously larger or smaller in one of the sample groups?
```